### ISSUE TITLE
Make the clickable area of the knob circular rather than the rectangle of the canvas.

### ISSUE THREAD

**Issue by s63** [aterrien/jQuery-Knob#86]
Currently if you click 2 pixels to the right of the knob, nothing happens because this area is outside the knob.

If you click 2 pixels diagonally up and right of the knob, the knob receives the click because the mouse input area is a square rather than a circle.

Would it be possible to have the mouse input area of the knob be a circle, so that any clicks in the transparent corners of the canvas fall through to the layer beneath?

Basically my problem is that I'm trying to use an input enabled version of the super-imposed example. It works fine if you click on the arc at 90º angles from the origin, but when clicking at 45º angles you end up changing the inner knob instead.
